cli/exec/exec.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/rs/zerolog/log"
 	"github.com/urfave/cli/v3"
 
+	"go.woodpecker-ci.org/woodpecker/v2/agent"
```

Comment:
if you need to import the agent then the const is in the wrong place and should be moved elsewhere ...!

I suggest [go.woodpecker-ci.org/woodpecker/v2/pipeline](https://github.com/woodpecker-ci/woodpecker/blob/main/pipeline/const.go)